esda/getisord.py

```diff
@@ -2,9 +2,9 @@
 Getis and Ord G statistic for spatial autocorrelation
 """
 __author__ = "Sergio J. Rey <srey@asu.edu>, Myunghwa Hwang <mhwang4@gmail.com> "
-__all__ = ['G', 'G_Local']
+__all__ = ["G", "G_Local"]
```

Comment:
Is there a specific reason for swapping out the single quotes for doubles quotes?

Reply:
black formatter